lib/Room.js

```diff
@@ -214,6 +216,24 @@ class Room extends EventEmitter
 		return this._protooRoom.peers.length;
 	}
 
+	async writeManager() {
+		if (writeStack.length > 0) {
+			const task = writeStack.reverse().pop();
```

Comment:
This looks like a bug? Reverse will change `writeStack`, so each time this is called we will flip the stack. Is that intended?

Reply:
bug, thanks, i should be making a temp var to flip 👍